wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/cli/class-export.php

```diff
@@ -102,6 +102,17 @@ public function __invoke( $args, $assoc_args ) {
 
 			$files = array_merge( $files, $jed_files );
 
+			// Create PHP file.
+			$php_file = "{$wp_locale}.l10n.php";
+			if ($context) {
+				$php_file = "$context-{$php_file}";
+			}
+			$php_file = "{$dest}/{$php_file}";
+			$result = $this->build_php_file( $gp_project, $gp_locale, $translation_set, $po_entries, $php_file );
+			if ($result) {
+				$files[] = $php_file;
```

Comment:
```suggestion
			// Create PHP file.
			$php_file = "{$wp_locale}.l10n.php";
			if ( $context ) {
				$php_file = "$context-{$php_file}";
			}
			$php_file = "{$dest}/{$php_file}";

			$result = $this->build_php_file( $gp_project, $gp_locale, $translation_set, $po_entries, $php_file );
			if ( $result ) {
				$files[] = $php_file;
```

Some nitpicky whitespace.